### PR TITLE
Jkmarx/add tools list landing page

### PR DESCRIPTION
**Pull request by jkmarx**
Ref #2680 

- Added tools list
Note: Just getting the components in and will focus on styling once all the pieces are in place.
screen shot 2018-11-28 at 2 17 40 pm

### CODE REVIEW

File: refinery/ui/source/js/home/ctrls/home-ctrl.spec.js [hunk 1]
```diff
@@ -1,24 +0,0 @@
-(function () {
```
Comment (jkmarx) [inline refinery/ui/source/js/home/ctrls/home-ctrl.spec.js]: unused

File: refinery/tool_manager/serializers.py [hunk 2]
```diff
@@ -68,6 +68,12 @@ class Meta:
 class ToolDefinitionSerializer(serializers.ModelSerializer):
     file_relationship = FileRelationshipSerializer()
     parameters = ParameterSerializer(many=True, allow_null=True)
+    workflow = serializers.SerializerMethodField()
+
+    def get_workflow(self, tool):
+        if tool.workflow:
+            return tool.workflow.uuid
+        return tool.workflow
```
Comment (scottx611x) [inline refinery/tool_manager/serializers.py]: Wouldn't the 2nd `return tool.workflow` always be `None`here?

Comment (jkmarx) [reply refinery/tool_manager/serializers.py]: Yes, am I being redundant?

Comment (scottx611x) [reply refinery/tool_manager/serializers.py]: Maybe a little. How about something like
```
def get_workflow(self, tool):
    return tool.workflow.uuid if tool.is_workflow() else None
```
or
```
def get_workflow(self, tool):
    if tool.is_visualization():
        return None  # Vis Tools have no associated Workflows
    return tool.workflow.uuid
```

Comment (jkmarx) [reply refinery/tool_manager/serializers.py]: I don't see how that improves anything.

**Pull request merged**